core/src/main/java/org/lflang/target/property/CompilerProperty.java

```diff
@@ -0,0 +1,12 @@
+package org.lflang.target.property;
+
+import java.util.List;
+import org.lflang.Target;
+
+public class CompilerProperty extends AbstractStringConfig {
+
+  @Override
+  public List<Target> supportedTargets() {
+    return Target.ALL;
```

Comment:
Is this property also supported by the TypeScript target?

Reply:
Nope. Will change this.